backend/src/appointment/routes/api.py

```diff
@@ -437,121 +437,6 @@ def read_public_appointment(slug: str, db: Session = Depends(get_db)):
     )
 
 
-@router.put("/apmt/public/{slug}", response_model=schemas.SlotAttendee, deprecated=True)
```

Comment:
Deprecated route, contains duplicate zoom meeting link code so removed it and its tests to avoid confusion.

Reply:
Yeah this is still from the single appointment feature we dropped. There might still be more around...

Reply:
There is! but they're marked as deprecated. I need to enable this and remove them and their tests at some point: https://github.com/thunderbird/appointment/issues/334